com.sap.cloud.lm.sl.cf.database/src/test/java/com/sap/cloud/lm/sl/cf/database/migration/executor/DatabaseSequenceMigrationExecutorTest.java

```diff
@@ -0,0 +1,75 @@
+package com.sap.cloud.lm.sl.cf.database.migration.executor;
+
+import java.sql.SQLException;
+
+import javax.sql.DataSource;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import com.sap.cloud.lm.sl.cf.database.migration.client.DatabaseQueryClient;
+
+class DatabaseSequenceMigrationExecutorTest {
+
+    private static final String TEST_SEQUENCE_NAME = "test";
+
+    private static final long ZERO_SEQUENCE_VALUE = 0L;
```

Comment:
You don't need constants for these. Just use them directly. Constants are useful for reducing duplication and giving a meaningful name to something, but here they're not doing either.